### PR TITLE
Added Ubuntu version 2004.2020.424.0

### PR DESCRIPTION
**Pull request by jedieaston**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

For the benefit of [our friends](https://github.com/microsoft/winget-pkgs/issues/26805#issuecomment-962691304) on pre-2004 Windows. This URL was fiddlered out of `wsl --install`, if the WSL team has a problem with this we can remove it.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34324)

### CODE REVIEW

File: manifests/c/Canonical/Ubuntu/2004.2020.424.0/Canonical.Ubuntu.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Canonical.Ubuntu
```
Comment (Trenly) [inline manifests/c/Canonical/Ubuntu/2004.2020.424.0/Canonical.Ubuntu.installer.yaml:4]:
```suggestion
PackageIdentifier: Canonical.Ubuntu.2004
```

File: manifests/c/Canonical/Ubuntu/2004.2020.424.0/Canonical.Ubuntu.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Canonical.Ubuntu
```
Comment (Trenly) [inline manifests/c/Canonical/Ubuntu/2004.2020.424.0/Canonical.Ubuntu.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Canonical.Ubuntu.2004
```

File: manifests/c/Canonical/Ubuntu/2004.2020.424.0/Canonical.Ubuntu.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Canonical.Ubuntu
```
Comment (Trenly) [inline manifests/c/Canonical/Ubuntu/2004.2020.424.0/Canonical.Ubuntu.yaml:4]:
```suggestion
PackageIdentifier: Canonical.Ubuntu.2004
```

**Review by Trenly — CHANGES_REQUESTED** [review]
~This package was moved~

Removed based on comments below

**Comment by jedieaston** [comment]
To where? This is a different package than Ubuntu.2004, with a different PackageFamilyName. They have the same version number, but that's just because the "default" Ubuntu package installs the latest LTS, which is 20.04.

**Comment by Trenly** [comment]
> To where? This is a different package than Canonical.Ubuntu.2004, with a different PackageFamilyName. They have the same version number, but that's just because the "default" Ubuntu package installs the latest LTS, which is 20.04.

Ohhh, I see. So this is just the latest LTS?

**Comment by jedieaston** [comment]
Yep.

**Review by Trenly — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**